pattoo/api/portal/react/src/PurgeForm.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+class PurgeForm extends React.Component {
```

Comment:
Where are the comments on what you are doing and why? This is important for future developers.